enterprise/app/code/code.tsx

```diff
@@ -647,7 +661,7 @@ export default class CodeComponent extends React.Component<Props, State> {
           head: this.state.repoResponse?.defaultBranch,
         })
       )
-      .then((response) => {
+      .then(async (response) => {
         console.log(response);
         let newCommits = response.aheadBy;
         let newSha = response.commits.pop()?.sha;
```

Comment:
not an issue with this PR specifically but FYI I think `newCommits == new Long(0)` will always return false on line 668 below - probably want either `Number(newCommits) === 0` or `Long.ZERO.equals(newCommits)`

Reply:
Done